components/table/table-wrapper.js

```diff
@@ -225,12 +220,31 @@ export class TableWrapper extends RtlMixin(SelectionMixin(LitElement)) {
 				display: none;
 			}
 			:host([type="light"]) {
+				--d2l-table-border-radius: 0rem; /* stylelint-disable-line length-zero-no-unit */
+				--d2l-table-border-radius-sticky-offset: 0rem; /* stylelint-disable-line length-zero-no-unit */
 				--d2l-table-cell-height: 1.15rem; /* min-height to be 48px including border */
 				--d2l-table-cell-padding: 0.6rem;
 				--d2l-table-cell-padding-alt: calc(0.6rem - 1px) 0.6rem 0.6rem 0.6rem;
 				--d2l-table-border-color: var(--d2l-color-gypsum);
 				--d2l-table-header-background-color: #ffffff;
 			}
+			:host([sticky-headers]) {
+				--d2l-table-controls-shadow-display: none;
+			}
+			.d2l-sticky-headers-backdrop {
+				position: sticky;
+				top: calc(var(--d2l-table-sticky-top) + var(--d2l-table-border-radius));
+				width: 100%;
+				z-index: 2; /* Must sit under d2l-table sticky-headers but over sticky columns and regular cells */
+			}
+			.d2l-sticky-headers-backdrop::after {
+				background-color: var(--d2l-table-controls-background-color, white);
```

Comment:
This needs a solid colour - I've used `--d2l-table-controls-background-color` with a fallback to `white`, but that's not bulletproof (e.g. `--d2l-table-controls-background-color` might be defined explicitly in the `d2l-table-controls` scope, which we wouldn't have access to here).